config/linux/build.xml

```diff
@@ -5,25 +5,63 @@
 <project name="native-linux" basedir="../.." xmlns:if="ant:if" xmlns:unless="ant:unless">
     <import file="../build-definitions.xml"/>
 
-    <condition property="LIB_POSTFIX" value="" else="32">
-        <equals arg1="${build.arch}" arg2="x64"/>
+    <condition property="build.arch.x86" value="true" else="false">
+        <equals arg1="${build.arch}" arg2="x86"/>
     </condition>
 
     <condition property="build.arch.x64" value="true" else="false">
         <equals arg1="${build.arch}" arg2="x64"/>
     </condition>
 
+    <condition property="build.arch.arm32" value="true" else="false">
+        <equals arg1="${build.arch}" arg2="arm32"/>
+    </condition>
+
+    <condition property="build.arch.arm64" value="true" else="false">
+        <equals arg1="${build.arch}" arg2="arm64"/>
+    </condition>
+
+    <condition property="LIB_POSTFIX" value="32">
+        <istrue value="${build.arch.x86}"/>
+    </condition>
+
+    <condition property="LIB_POSTFIX" value="arm32">
+        <istrue value="${build.arch.arm32}"/>
+    </condition>
+
+    <condition property="LIB_POSTFIX" value="arm64">
+        <istrue value="${build.arch.arm64}"/>
+    </condition>
+
+    <condition property="LIB_POSTFIX" value="">
```

Comment:
In the rest of this file the word suffix is used instead, e.g. Line 63&64.
Might be good to have some consistency.

Reply:
Tentative plan (as discussed in slack) was to remove postfix entirely and do per-folder archs. (Similar to android)